openreview/venue_request/process/revisionProcess.py

```diff
@@ -19,6 +19,13 @@ def process(client, note, invitation):
 
         if invitation_type == 'Revision':
             conference.create_submission_stage()
+            abstract_deadline = forum_note.content.get('abstract_registration_deadline', '').strip()
```

Comment:
instead of parsing the date again, you can get the value from the SubmissionStage object, right?

conference.submission_stage.second_due_date o conference.submission_stage.duedate

Reply:
I didn't know that. I will look into `second_due_date` and `due_date`, I'm a bit confused by them. I thought they were the submission deadline, how can it be the abstract?

Reply:
@celestemartinez do you mind to clarify how is the mapping of the abstract deadline with the second_due_date and due_date values?

you can find the login here: https://github.com/openreview/openreview-py/blob/master/openreview/conference/helpers.py#L99

I think the matching invitation cdate should always match with the due_date that for venues with one single deadline that is the full paper deadline and for venue with two deadlines that is the abstract deadline.

Reply:
I know this is kind of confusing, but this is how we are getting the dates from the request form:

if the venue has abstract and submission deadline, then:
- `submission_stage.due_date = request_form.content['abstract_registration_deadline']`
- `submission_stage.second_due_date = request_form.content['Submission Deadline']`

if the venue only has a submission deadline, then:
- `submission_stage.due_date = request_form.content['Submission Deadline']`
- `submission_stage.second_due_date = None`

So @melisabok  is correct: we should always use `submission_stage.due_date`, because for venue with one deadline this represents the submission deadline, and for venues with two deadlines this represents the abstract deadline.

Reply:
Thank you Celeste, I understand these dates now!